### DISCUSSION TITLE
Is it possible to increase the gene label size in Combined Results Graph?

### DISCUSSION THREAD

**Discussion by fengx90** [egeulgen/pathfindR#116] [Q&A] [answered]
Hello Ege,

Thanks very much for developing a great package for the community! 
When I plot the enriched terms, the labeling of genes and term descriptions are quite small. Is there a way to increase the labeling size for better output figure quality? Thanks very much!

Best,
Feng

**Comment by YunielFM** [comment, marked as answer]
Hi Feng, the answer is quite easy. The results from the enrichment plots are actually ggplots. This mean that you can manually customize most of them.  Example:
1. Create the bubble chart from your `pathfindR` result, in this case, show 40 pathways.
`p <- enrichment_chart(pathwaysT1, top_terms = 40)`

2. Now you can customize everything in the `ggplot` object, in this case, I am reducing the size of the text on the y-axis to accommodate all the labels. You would need to do the opposite.
 `p +  theme(axis.text.y = element_text(size = 7))`

Happy plotting, and indeed, the package rocks!

**Reply by YunielFM** [reply]
For the sake of completeness, if you were referring to the **network plots**, the text size adjustment is a  bit tricky but doable. You basically need to hack into the function and add the text size you want for each gene in the plot.
Just type in your console `combined_results_graph`. Copy the output and create a customized function , smth like `my.combined.plot <- function (combined_df, selected_terms = "common",..........`. Then, in  **your function** change the line:
`p <- p + suppressWarnings(ggraph::geom_node_text(ggplot2::aes_(label = ~name),
                                                                                                   nudge_y = 0.2,
                                                   repel = TRUE, 
                                                   max.overlaps = 20))`
 to:
`p <- p + suppressWarnings(ggraph::geom_node_text(ggplot2::aes_(label = ~name),
                                                   size = 3, 
                                                   color = "grey0",
                                                   nudge_y = 0.2,
                                                   repel = TRUE, 
                                                   max.overlaps = 20))`
Notice I also changed the text color label. Probably there is another way, but this will solve your issue. 
Of course, use **your function** to create the network plot!!

**Reply by fengx90** [reply]
Thanks very much for the reply! This is actually what I want, not to change the axis text size. Gonna try out!

**Reply by fengx90** [reply]
Just tried and it works! Thanks again! Really appreciated it!

**Reply by YunielFM** [reply]
Excellent!
